dali/benchmark/normal_distribution_gpu_bench.cc

```diff
@@ -42,6 +42,7 @@ BENCHMARK_DEFINE_F(OperatorBench, NormalDistributionGPU_NonUniform)(benchmark::S
   TensorListShape<3> shape(batch_size);
   int vol_log2 = 0;
   for (int i = 1; i < max_sample_vol; i *= 2) vol_log2++;
+  assert(vol_log2 > 0);
```

Comment:
I don't think it is recognized by the tool as a value check as it happens only in the debug build.

Reply:
I know, but it serves as a debug check and as documentation saying that this value must be bigger than zero. We just need to mark it as intentional in Coverity